### PR TITLE
Cylc to scheduler

### PR DESCRIPTION
**Pull request by datamel**
Small PR, will fix the doc build after https://github.com/cylc/cylc-flow/pull/3907 goes in.
Changes references from [cylc] to [scheduler].

### CODE REVIEW

File: cylc/doc/etc/tutorial/consolidation-tutorial/flow.cylc [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[cylc]
+[scheduler]uler]
```
Comment (MetRonnie) [inline cylc/doc/etc/tutorial/consolidation-tutorial/flow.cylc:1]: Oops 😄

**Review by MetRonnie — CHANGES_REQUESTED** [review]

File: cylc/doc/etc/tutorial/consolidation-tutorial/flow.cylc (hunk 1 above)
Comment (datamel) [reply cylc/doc/etc/tutorial/consolidation-tutorial/flow.cylc:1]: Woops, no idea how I missed these :)

**Comment by MetRonnie** [comment]
Test failure is due to using old version of `actions/setup-node`, #174 should fix it

File: cylc/doc/etc/tutorial/consolidation-tutorial/flow.cylc [hunk 2]
```diff
@@ -1,4 +1,4 @@
-[cylc]
+[scheduler]]
```
Comment (MetRonnie) [inline cylc/doc/etc/tutorial/consolidation-tutorial/flow.cylc:1]: Still not quite right

**Review by MetRonnie — CHANGES_REQUESTED** [review]

File: cylc/doc/etc/tutorial/consolidation-tutorial/flow.cylc (hunk 2 above)
Comment (wxtim) [reply cylc/doc/etc/tutorial/consolidation-tutorial/flow.cylc:1]:
```suggestion
[scheduler]
```

Comment (datamel) [reply cylc/doc/etc/tutorial/consolidation-tutorial/flow.cylc:1]: yikes, not sure what is wrong with me!

**Review by MetRonnie — APPROVED** [review]

**Review by wxtim — APPROVED** [review]
✅

**Comment by MetRonnie** [comment]
❔

**Pull request merged**